Specs/Scene/PostProcessStageLibrarySpec.js

```diff
@@ -422,60 +415,52 @@ describe(
     });
 
-    // TODO: rewrite this test using loadAndZoomToModel
-    xit("bloom", function () {
+    it("bloom", function () {
       const origin = Cartesian3.fromDegrees(-123.0744619, 44.0503706, 100.0);
       const modelMatrix = Transforms.headingPitchRollToFixedFrame(
         origin,
         new HeadingPitchRoll()
       );
 
-      const model = scene.primitives.add(
-        Model.fromGltf({
+      return loadAndZoomToModel(
+        {
           url: boxTexturedUrl,
+          incrementallyLoadTextures: false,
           modelMatrix: modelMatrix,
-          scale: 40.0,
-        })
-      );
-
-      let ready = false;
-      model.readyPromise.then(function () {
-        ready = true;
-      });
-
-      const offset = new Cartesian3(
-        -37.048378684557974,
-        -24.852967044804245,
-        4.352023653686047
-      );
-      scene.camera.lookAt(origin, offset);
+        },
+        scene
+      ).then(function () {
+        scene.camera.lookAt(origin, new HeadingPitchRange(0.0, 0.0, 1.0));
 
-      return pollToPromise(function () {
-        scene.render();
-        return ready;
-      }).then(function () {
+        // Render without bloom
+        let originalColor;
         expect(scene).toRenderAndCall(function (rgba) {
+          originalColor = rgba;
           for (let i = 0; i < rgba.length; i += 4) {
             expect(rgba[i]).toBeGreaterThan(0);
-            expect(rgba[i + 1]).toEqual(0);
-            expect(rgba[i + 2]).toEqual(0);
+            expect(rgba[i + 1]).toBeGreaterThan(0);
+            expect(rgba[i + 2]).toBeGreaterThan(0);
             expect(rgba[i + 3]).toEqual(255);
           }
+        });
 
-          scene.postProcessStages.bloom.enabled = true;
-          scene.renderForSpecs();
-          expect(scene).toRenderAndCall(function (rgba2) {
-            for (let i = 0; i < rgba.length; i += 4) {
-              expect(rgba2[i]).toBeGreaterThan(0);
-              expect(rgba2[i + 1]).toEqual(0);
-              expect(rgba2[i + 2]).toEqual(0);
-              expect(rgba2[i + 3]).toEqual(255);
-
-              expect(rgba2[i]).not.toEqual(rgba[i]);
-            }
-          });
-          scene.postProcessStages.bloom.enabled = false;
+        // Render with bloom and compare
+        const bloom = scene.postProcessStages.bloom;
+        bloom.enabled = true;
+        // increase the brightness to make the difference more
+        // noticeable
+        bloom.uniforms.brightness = 0.5;
+        scene.renderForSpecs();
+        expect(scene).toRenderAndCall(function (rgba) {
+          expect(rgba).not.toEqual(originalColor);
+          for (let i = 0; i < rgba.length; i += 4) {
+            expect(rgba[i]).toBeGreaterThan(0);
+            expect(rgba[i + 1]).toBeGreaterThan(0);
+            expect(rgba[i + 2]).toBeGreaterThan(0);
+            expect(rgba[i + 3]).toEqual(255);
+          }
         });
+        scene.postProcessStages.bloom.enabled = false;
```

Comment:
Maybe this should go in an `afterEach` function? (same for the `"ambient occlusion"` test)

Reply:
These lines were already in `afterEach`, so I removed these redundant lines.